vhost-device-vsock/vhost-device-vsock.1

```diff
@@ -0,0 +1,149 @@
+.Dd July 24, 2024
+.Dt VHOST-DEVICE-VSOCK 1
+.Os
+.Sh NAME
+.Nm vhost-device-vsock
+.Nd A virtio-vsock device using the vhost-user protocol
+.Sh SYNOPSIS
+.Nm
+.Op OPTIONS
+.Fl \-socket Ar SOCKET
+.Fl \-uds-path Ar UDS_PATH
+.Sh DESCRIPTION
+The
+.Nm
+utility launches a daemon listening to
+.Ar SOCKET
+for incoming connections from
+.Em vhost-user
+front-ends.
+Upon successful connection and protocol negotiation,
+it starts a vhost-user backend instance.
+The functionality of a VIRTIO vsock device is provided
+using the specified unix socket,
+.Ar UDS_PATH Ns
+, to which a host-side application connects to.
+.sp
+.Bl -tag -width Ds
+.It Fl -help | h
+Print help information.
+.It Fl -version | V
+Print version information.
+.It Fl \-guest-cid Pa GUEST_CID
+Context identifier of the guest which uniquely identifies the
+device for its lifetime [default: 3]
```

Comment:
```suggestion
device for its lifetime
.Pq default: 3
```

for `(default: 3)` or 

```suggestion
device for its lifetime
.Bq default: 3
```
for `[default: 3]`

https://mandoc.bsd.lv/man/mdoc.7.html#Physical_enclosures